### PR TITLE
docs: add about/faq

### PR DESCRIPTION
**Pull request by eckartal**
## Describe Your Changes

Created a FAQ section. 
-

## Fixes Issues

- Closes #
- Closes #

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

**Review by 0xSage — DISMISSED** [review]
Gentle reminder, can we: 
- Move FAQ under /about/FAQ
- Have each question be a dropdown

I think @hieu-jan can help if you have questions!

File: docs/docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Yes, Jan can run locally without an internet connection for many features.
+
+## Is Jan compatible with my operating system?
+
+Jan is available for Mac, Windows, Linux, ensuring wide compatibility.
+
+## Are there any costs associated with using Jan?
+
+Jan is free to use. However, if you opt to connect to remote APIs, you will need to cover those services' costs according to their pricing, not Jan's.
```
Comment (0xSage) [inline docs/docs/faq.md:17]: Do be a bit more specific and concise.
"If you want to connect to remote APIs, like GPT-4, you will need to put in your own API key."

File: docs/docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Contributions can be made through GitHub and Discord, where you can also suggest features and contribute.
+
+## How can I get involved with the Jan community?
+
+Joining [Jan's Discord server](https://discord.gg/qSwXFx6Krr) is a great way to get involved with the community.
+
+## How do I troubleshoot issues with installing or using Jan?
+
+For troubleshooting, you should reach out on Discord and check GitHub for assistance and support from the community and the development team.
```
Comment (0xSage) [inline docs/docs/faq.md:37]: Hey I think this is missing some FAQs I added in the very beginning. Can we bring those back?

i.e. 

## How does Jan make money?
## Do you sell my data?

etc

Comment (eckartal) [reply docs/docs/faq.md:37]: Heyhey! I've added some basic questions you already added:

- What is Jan?
- Do you use my data?
- What does "Jan" stand for?

And also: 

- How do I troubleshoot issues when installing or using Jan?
- Can I self-host?
- Are you hiring?

etc.,

I've also sorted them by frequency of being asked.

**Comment by eckartal** [comment]
I am getting used to doing something on GitHub and love it! 
Thank you for your help @hieu-jan and @0xSage <3

**Comment by hieu-jan** [comment]
@eckartal  have updated the content. I will display it under about/faq. Because current question is very short with also short answer. I think we just put all the questions into one page

**Review by hieu-jan — APPROVED** [review]
New changes fixed all conservations!

**Pull request merged**